src/features/course-player/sequence-container/SequenceContainer.scss

```diff
@@ -297,6 +297,15 @@
   }
 }
 
+// Customize bottom navigation buttons
+.unit-navigation {
+
+  .btn-outline-secondary,
+  .btn-outline-primary {
+    width: 144px;
```

Comment:
Why the buttons have static width? What was wrong with the current approach? the width have to adapt, doing things like this increase the maintenance of a code if something change.